api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
@mattdrayer , @efagin Does `stageVariables.enterprise_host` contains the host name for api manager ? 
or is it the host where enterprise API is deployed ?